server/app/controllers/admin/AdminApplicationController.java

```diff
@@ -2,6 +2,7 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static views.admin.programs.ProgramApplicationView.NEW_STATUS;
+import static views.admin.programs.ProgramApplicationView.NOTE;
```

Comment:
Leaving comment here for general PR description: This doesn't address all of the open TODOs in #3020, so I'd recommend not marking it as fixing the issue. For example, the changes in PR #3306 will be required first.

In that PR, I repointed the "show the current note" TODOs against #3264, so I'd also suggest not marking this as fixing that issue. I couldn't find an issue in the status tracking tracker that was along the lines of "show the current note" that would be appropriate to point it against. I'd honestly just use #3264 for that, since setting an application note and being able to observe the changes seems like the smallest useful unit for a user.

Totally fine as a separate PR if you'd prefer, I'd just suggest not marking this PR as fixing the issues since there's still remaining work.

Reply:
I don't believe having Fixes closes the Issue, It's more as "This addresses issue X".

AFAIK our Fixes section is just self-imposed organization we have created similar to the Release Notes section and the checklist.  It's meaningless aside from humans who read it.

Reply:
I don't think that's quite correct. Empirically, GitHub closes issues prefixed with the "Fixes" keyword, I've made use of this in many PRs. To just reference another issue, not prefixing with "Fixes" does the trick (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

I've observed this behavior with PRs. See https://github.com/civiform/civiform/pull/3289 for an example where the issue was automatically marked as fixed once the merge had completed and https://github.com/civiform/civiform/pull/3207 for an example where the issue didn't have the "Fixes" prefix and was still referenced (but not closed).

Reply:
Ah confusing.  Seemingly you can't get it right if two parallel PRs address an Issue and collectively fix it.

updated